docs/source/mapping/basic-mapping.rst

```diff
@@ -0,0 +1,392 @@
+
+After doing this, the RePak folder should look like this:
+
+.. code-block:: text
+
+  RePak
+  ├──assets
+  |  └─texture
+  |    └─loadscreens
+  |      └─test_loadscreen.dds
+  ├──maps
+  |  └─loadscreen.json
+  ├──rpaks
+  ├──pack_all.bat
+  └─RePak.exe
+
+If you've followed everything so far, double click on ``pack_all.bat``. This will open a command prompt with some information given to you, which for this case you can simply skip past by hitting any key on your keyboard. 
+
+After you've done this and RePak has created the ``.rpak``, you should see a file called ``mp_test_loadscreen.rpak`` in the ``rpaks`` folder of RePak.
+
+Adding the loadscreen to the Northstar mod (updated mod format):
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Create a folder called ``paks`` in the root of your mod's directory. Inside of here, place the ``mp_test_loadscreen.rpak`` file that we exported from RePak earlier and create a file called ``rpak.json``. Open ``rpak.json`` in a text editor and put in the text shown below:
+
+.. code-block:: text
+
+	{
+		"Preload": {
+			"mp_test_loadscreen.rpak": "false"
+		}
+	}
+
+If this doesn't work, make sure the names match up properly, that the image is ``1920`` x ``1080``, and that you used compression method ``DXT1`` when exporting the image from your image editor.
+
+Testing the loadscreen:
+^^^^^^^^^^^^^^^^^^^^^^^
+
+If you've followed everything to this point, your mod folder should look something like this:
+
+.. code-block:: text
+
+  Testmap
+  ├──mod.json
+  ├──mod
+  |  ├──materials
+  |  |  └─world
+  |  |    ├──test.vmt
+  |  |    └─test.vtf
+  |  └─maps
+  |    ├──mp_test.map
+  |    └─mp_test.bsp
+  └─paks
+    ├──mp_test_loadscreen.rpak
+    └─rpak.json
+
+After all of this, the map should have a loading screen when loading it as we did before. 
+
+Continuing to create the map:
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+From here, you might be able to piece together bits and pieces of information in order to create a proper functioning map to your liking. Due to the simplicity of this guide, you probably won't be able to go straight from this to a full size map found in vanilla Titanfall 2, but you should at least know the basics. You can always ask for more help in the ``#maps-chat`` channel in the Northstar Discord if you're confused.
+
+Other:
+------
+
+Netradiant Custom Tutorial - Part 2:
+
+..  youtube:: JZO8H4rBqtA
+
+How to make corners correctly
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Improper corners can cause the player to get stuck
+
+.. figure:: /_static/map-corners.png
+  :class: screenshot
+
+..  youtube:: mwvxonuCm8U
+
+Sample clip: 
+
+..  youtube:: zgWDme7Y6oI
+
+Sample map: https://cdn.discordapp.com/attachments/925435799057604709/1041813222547791953/corner_test_map.map
```

Comment:
Discord cdn links expire now. I have it available on my server if nobody has anything against that
```suggestion
`Sample map <https://salzgrube.club/static/northstar/corner_example.map>`_
```

Reply:
No problems with that personally, but github's cdn may be better?

Reply:
Putting the file in the repo might be a better idea then, I wouldn't bet on github's cdn keeping everything forever

Reply:
What's the update on this?

Reply:
idk flip a coin or smth